authors/settings.py

```diff
@@ -11,6 +11,9 @@
 """
 
 import os
+import dj_database_url
+
+import dj_database_url
```

Comment:
Why do we have all this duplication

Reply:
I have worked that. Thank you for pointing that out.